### ISSUE TITLE
sigmoid vs softmax

### ISSUE THREAD

**Issue by jaysharp** [Nirvanada/Deep-and-Cross-Keras#3]
In the paper of DCN, sigmoid is used instead of softmax.
so could you kindly share your point of view regarding that?
